### PR TITLE
Contracts - draft - do not merge

### PR DESCRIPTION
**Pull request by PatAltimore**

### CODE REVIEW

File: api-reference/beta/api/contract_get.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Get Contract
+
+Retrieve the properties and relationships of [contract](../resources/contract.md) object.
```
Comment (dkershaw10) [inline api-reference/beta/api/contract_get.md]: I think we need more details here.  From the AAD Graph docs, for which I think you can tidy up some of the language and include here:
Exists in Partner Tenants only. Partner Tenants are Azure AD tenants that belong to Microsoft Partners who are either part of Office 365 Syndication, Microsoft Cloud Solution Provider, or Microsoft Advisor partner programs. Represents an existing partnership that the Partner Tenant has with a Customer Tenant. Read-only.

File: api-reference/beta/resources/contract.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Contract resource type
+Represents an existing partnership that the partner tenant has with a customer tenant.
+> Exists in partner tenants only. Partner tenants are Azure AD tenants that belong to Microsoft partners who are either part of Office 365 Syndication, Microsoft Cloud Solution Provider, or Microsoft Advisor partner programs.
```
Comment (dkershaw10) [inline api-reference/beta/resources/contract.md]: Let's put Cloud Solution Provider first AND make this a link to:  https://partnercenter.microsoft.com/en-us/partner/programs

File: api-reference/beta/resources/contract.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Contract resource type
+Represents an existing partnership that the partner tenant has with a customer tenant.
+> Exists in partner tenants only. Partner tenants are Azure AD tenants that belong to Microsoft partners who are either part of Office 365 Syndication, Microsoft Cloud Solution Provider, or Microsoft Advisor partner programs. 
+
+### Methods
+
+| Method		   | Return Type	|Description|
+|:---------------|:--------|:----------|
+|[Get contract](../api/contract_get.md) | Contract |Read properties and relationships of contract object.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/contract.md]: Read properties of a specific contract object.  (There are no relationships on contracts)

File: api-reference/beta/resources/contract.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+|[List contracts](../api/contract_list.md) | Contract collection | List of contracts in the partner tenant. |
+
+### Properties
+| Property	   | Type	|Description|
+|:---------------|:--------|:----------|
+|contractType|String|Type of the contract.</br></br>Possible values are:</br> *SyndicationPartner*: Partner that exclusively resells and manages O365 and Intune for this customer. They resell and support their customers.</br> *BreadthPartner*: Partner has the ability to provide administrative support for this customer. However, the partner is not allowed to resell to the customer.</br>*ResellerPartner*: Partner that is similar to a syndication partner, except that the partner doesn’t have exclusive access to a tenant. **(pa) Should this be removed or moved to SyndicationPartner? It's confusing ->** In the syndication case, the customer cannot buy additional direct subscriptions from Microsoft or from other partners.|
+|customerId|Guid|The unique identifier for the customer tenant referenced by this partnership. Corresponds to the id property of the customer tenant's organization resource. |
```
Comment (dkershaw10) [inline api-reference/beta/resources/contract.md]: I don't have a problem with the description.  If you want to reword, that's OK.  Also double checking that these are accurate.

File: api-reference/beta/resources/contract.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+|defaultDomainName|String|A copy of the customer tenant's default domain name. The copy is made when the partnership with the customer is established. It is not automatically updated if the customer tenant's default domain name changes.|
+|displayName|String|A copy of the customer tenant's display name. The copy is made when the partnership with the customer is established. It is not automatically updated if the customer tenant's display name changes.|
+|id|String| The unique identifier for the partnership. Inherited from [directoryObject](directoryobject.md). Read-only.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/contract.md]: Let's remove the Inherited from thing.  Don't think this adds much.

File: api-reference/beta/resources/contract.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+### Methods
+
+| Method		   | Return Type	|Description|
+|:---------------|:--------|:----------|
+|[Get contract](../api/contract_get.md) | Contract |Read properties and relationships of contract object.|
+|[List contracts](../api/contract_list.md) | Contract collection | List of contracts in the partner tenant. |
+
+### Properties
+| Property	   | Type	|Description|
```
Comment (dkershaw10) [inline api-reference/beta/resources/contract.md]: Not sure if we want to say this, but contractType, customerId, defaultDomainName, and displayName are all filterable.  Maybe say that in the List topic?

File: api-reference/beta/resources/contract.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+|customerId|Guid|The unique identifier for the customer tenant referenced by this partnership. Corresponds to the id property of the customer tenant's organization resource. |
+|defaultDomainName|String|A copy of the customer tenant's default domain name. The copy is made when the partnership with the customer is established. It is not automatically updated if the customer tenant's default domain name changes.|
+|displayName|String|A copy of the customer tenant's display name. The copy is made when the partnership with the customer is established. It is not automatically updated if the customer tenant's display name changes.|
+|id|String| The unique identifier for the partnership. Read-only.|
```
Comment (AlanWong-MS) [inline api-reference/beta/resources/contract.md]: Is there a special font or format for indicating that the "id" is the key of the entity? I see that the list is alphabetized, but just wondering if it improves UX if the key "id", which happens to be placed at the bottom, is outlined differently. Not trying to push for this or anything--just an idle thought.

Comment (PatAltimore) [inline api-reference/beta/resources/contract.md]: Thanks for pointing it out.  We usually state it's the key in the description.  I'll add that.

**Comment by PatAltimore** [comment]
New PR opened against staging. https://github.com/microsoftgraph/microsoft-graph-docs/pull/905

**Pull request closed without merging**